src/data/CCBV.ml

```diff
@@ -1,80 +1,117 @@
 
-(* This file is free software, part of containers. See file "license" for more details. *)
-
 (** {2 Imperative Bitvectors} *)
 
-let __width = Sys.word_size - 2
+let __width = Sys.word_size - 1
+
+(** We use OCamls ints to store the bits. We index them from the
+    least significant bit. We create masks to zero out the most significant
+    bits that aren't used to store values. *)
+let __lsb_masks =
```

Comment:
I see. Let's keep things simple for now, if you agree — we can add a version with memoization in the benchmarks file and play with it later.